client/app/Streamfinder.jsx

```diff
@@ -66,7 +86,8 @@ class Streamfinder extends React.Component {
             </Route>
             <Route path="/search">
               <ErrorBoundary>
-                <Search />
+                //search prolly only needs to update most recent id searched
+                <Search switch={this.handleSearchIdSwitch} />
               </ErrorBoundary>
```

Comment:
Is this how I would add the error boundary to my component?

Reply:
Yes.